packages/web-app-files/src/mixins/resourceCleanup.js

```diff
@@ -0,0 +1,69 @@
+import { mapState } from 'vuex'
```

Comment:
I'm still wondering if it's really faster to have this mixin instead of getters in the store.

Reply:
Apart from that: the mixin name is weird. ;-)

Reply:
yup, i already forgot that we still had an ongoing discussion regarding that, i try to get it out